docs/PLUGINS.md

```diff
@@ -233,11 +244,12 @@ some changes made to your plugin break this configuration model.
 
 ### Backward compatibility test
 
-About the later, in case you need to introduce some breaking changes, you can define a backward compatibility test case
+About the latter, in case you need to introduce some breaking changes, you can define a backward compatibility test case
 
-```yaml
-    @Test public void should_be_backward_compatible() throws Exception {
-        ConfigurationAsCode.get().configure(ConfigAsCodeTest.class.getResource("obsolete-configuration-as-code.yml").toString());
+```java
+    @Test 
+    @ConfiguredWithCode("obsolete-configuration-as-code.yml")
```

Comment:
wouldn't this require that you set JCasC's depreciation to be allowed instead of fail?

Reply:
its there 2 lines down after this block

Reply:
I must be blind :flushed: